apps/aptos/components/Farms/hooks/useCheckIsUserIpPass.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@pancakeswap/awgmi'
+import { CHECK_USER_IP_API } from 'config/index'
+
+export const useCheckIsUserIpPass = (): boolean => {
+  const { data } = useQuery(['checkIsUserIpPass'], async () => {
```

Comment:
Let's check when user is connected. and make it not revalidate that across the session